ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thanks for using Certbot - if you're experiencing issues with using Certbot
+we'd appreciate it if you could answer a few questions about your environment so
+that we can better assist you:
+
+## My operating system is (include version):
+
+
+## My web server is (include version):
+
+
+## How did you install Certbot:
+
+
+## What command did you run? What output did it produce?
```

Comment:
[supernit](http://www.baseballnationtx.com/wp-content/uploads/2016/12/2017_SUPER-NIT_banner.jpg): I personally like the wording "What command did you run and what output did it produce?" better. If you like it better this way, we can leave it.